fundamentals-review.livemd

```diff
@@ -239,45 +283,47 @@ Remember that you can use `List` functions such as `List.insert_at/3` with keywo
 </details>
 
 ```elixir
-[a: 1, c: 3]
+List.insert_at([a: 1, c: 3], 1, b: 2)
```

Comment:
This is a tricky question.

The following are equivalent:
```elixir
List.insert_at([a: 1, c: 3], 1, b: 2)
List.insert_at([a: 1, c: 3], 1, [b: 2])
```

Remember that keyword lists are lists of `{:atom, term}` tuples, so you need to insert a tuple.

```elixir
List.insert_at([a: 1, c: 3], 1, {:b, 2})
```